### PR TITLE
Added Example to OCPP 2.0.1

### LINKED ISSUE #142: Create examples for OCPP 2.0.1

### PR DESCRIPTION
**Pull request by tropxy**
Also fixed subprotocol handling in all examples

fixes #142

### CODE REVIEW

File: examples/v16/central_system.py [hunk 1]
```diff
@@ -31,10 +31,22 @@ def on_boot_notitication(self, charge_point_vendor: str, charge_point_model: str
 
 
 async def on_connect(websocket, path):
-    """ For every new charge point that connects, create a ChargePoint instance
-    and start listening for messages.
-
+    """ For every new charge point that connects, create a ChargePoint
+    instance and start listening for messages.
     """
+    requested_protocols = websocket.request_headers['Sec-WebSocket-Protocol']
```
Comment (OrangeTux) [inline examples/v16/central_system.py:37]: The bounds of `websocket.request_headers` aren't checked here. A client that doesn't send the header 'Sec-WebSocket-Protocol' will cause a `KeyError` here.

File: examples/v16/central_system.py [hunk 2]
```diff
@@ -31,10 +31,22 @@ def on_boot_notitication(self, charge_point_vendor: str, charge_point_model: str
...
+    if bool(websocket.subprotocol):
+        logging.info("Protocols Matched: %s", websocket.subprotocol)
```
Comment (OrangeTux) [inline examples/v16/central_system.py:39]: Wouldn't `if websocket.subprotocol == '2.0.1` be more explicit?

**Review by OrangeTux — CHANGES_REQUESTED** [review]

File: examples/v16/central_system.py (hunk 2 above)
Comment (tropxy) [reply examples/v16/central_system.py:39]: For this example in specific, where I had used just one supported lib, yes. But in general, you can support several, so if the result of the subprotocol is positive, then you can continue with the communication, regardless of which subprotocol of the list you have chosen

File: examples/v16/central_system.py (hunk 1 above)
Comment (tropxy) [reply examples/v16/central_system.py]: Agreed 👍

File: examples/v16/central_system.py (hunk 2 above)
Comment (OrangeTux) [reply examples/v16/central_system.py]: Ok. Fair point.

**Pull request merged**